src/Common/src/TypeSystem/Common/MetadataVirtualMethodAlgorithm.cs

```diff
@@ -586,5 +594,14 @@ public static IEnumerable<MethodDesc> EnumAllVirtualSlots(MetadataType type)
                 } while (type != null);
             }
         }
+
+        public override IEnumerable<MethodDesc> ComputeAllVirtualMethods(TypeDesc type)
+        {
+            foreach (var method in type.GetMethods())
+            {
+                if (method.IsVirtual)
+                    yield return method;
+            }
```

Comment:
Note that this code isn't really related to the rest of the code in this algorithm.